client/src/routes/account/components/programForm/form.js

```diff
@@ -288,6 +288,15 @@ class ProgramForm extends React.Component {
                                      value={values.yearLevels}
                                      options={this.optionsYearLevels}
                   />
+                  <input type="checkbox" className="form-check form-check-inline" onClick={(event) => {
+                    const selectAll = ['K', '1', '2', '3', '4', '5', '6', '7', '8', '9', '10', '11', '12'];
+                    event.target.checked
```

Comment:
This is the right logic, but we can make it more Reacty

Reply:
@elisechant, now have a separate component ...